tests/xtro-sharpie/EnumCheck.cs

```diff
@@ -57,8 +57,18 @@ public override void VisitEnumDecl (EnumDecl decl, VisitKind visitKind)
 			if (!enums.TryGetValue (mname, out var type)) {
 				Log.On (framework).Add ($"!missing-enum! {name} not bound");
 				return;
-			} else
-				enums.Remove (mname);
+			} else {
+            	var enumMembersInXamarin = type.Fields.Select (e => name.Replace ("Code","") + e.Name);
+            	var enumMembersInNewAppleLib = decl.Values.Select (e => e.Name);
+            	var missingEnums = enumMembersInNewAppleLib.Except (enumMembersInXamarin);
+            	if (missingEnums.Any ()) {
+            		foreach (var missingEnum in missingEnums) {
+            			Log.On (framework).Add ($"!missing-enum-member! {name} : {missingEnum.Replace (name.Replace ("Code",""),"")} not bound");
+            		}
+            		return;
+            	}
+            	enums.Remove (mname);
+            }
```

Comment:
Not critical but your indentation is incorrect :)

Now to be really useful the logic needs to be a bit more complex. In contrast to API you can always use a missing value, e.g. `Call ((MyEnum)42))`, so a missing member is not critical (as much as other things) since it's not blocking (just annoying). Also adding them later is not a breaking change...

The most important thing for enum members is their values, e.g. if we have `MissingMember = 43` while the right value is `42`. That's very hard to diagnose such errors (and easy to get wrong since, most of the time, the values are assigned automatically by the C# compiler). It's also a breaking change to fix them.

Reply:
Thanks for the feedback, I will try to add the value logic.

Reply:
I'm having a hard time finding a way to get the enum member **value** of Apple lib. 
The enum is of type ```Clang.Ast.EnumDecl```and the enum members type is ```Clang.Ast.EnumConstantDecl```. I didn't find anything online plus I can't find it by debugging. If someone have an idea I would be happy to know about it :)